modules/util/source-formatter/src/main/resources/documentation/all_checks.markdown

```diff
@@ -114,7 +114,7 @@ IllegalImportsCheck | [Bug Prevention](bug_prevention_checks.markdown#bug-preven
 IllegalTaglibsCheck | [Bug Prevention](bug_prevention_checks.markdown#bug-prevention-checks) | .ftl, .jsp, .jspf, .jspx, .tag, .tpl or .vm | Finds cases of incorrect use of certain taglibs. |
 [IncorrectFileLocationCheck](check/incorrect_file_location_check.markdown#incorrectfilelocationcheck) | [Bug Prevention](bug_prevention_checks.markdown#bug-prevention-checks) | | Checks that `/src/*/java/` only contains `.java` files. |
 IncorrectFilePathCheck | [Bug Prevention](bug_prevention_checks.markdown#bug-prevention-checks) | | Checks that file path contains illegal characters. |
-[InstanceInitializerCheck](check/instance_initializer_check.markdown#instanceinitializercheck) | [Styling](styling_checks.markdown#styling-checks) | .java | Performs several checks on class instance initializer. |
```

Comment:
@MicaelleSilvaa renegerate the documentation, please.

Reply:
resolved